### PR TITLE
Bug 1760858: Add special Edge drop shadow

### PR DESCRIPTION
**Pull request by andrewballantyne**
Fixes the following issues:

- https://jira.coreos.com/browse/ODC-2186 (node is rendered broken) - also the linked bugzilla
- https://jira.coreos.com/browse/ODC-1575 (can't see build progress)
- https://jira.coreos.com/browse/ODC-1576 (can't see app name)

### Edge Before

EdgeBrokenTopology

### Edge After

EdgeUpdatedTopology2

### Chrome

image

### Firefox

image

### Safari

image

### CODE REVIEW

**Comment by andrewballantyne** [comment]
FYI @openshift/team-devconsole-ux @jeff-phillips-18 @christianvogt

File: frontend/packages/dev-console/src/components/svg/SvgDropShadowFilter.tsx [hunk 1]
```diff
@@ -19,24 +19,48 @@ const SvgDropShadowFilter: React.FC<SvgDropShadowFilterProps> = ({
...
-        stdDeviation={stdDeviation}
-        floodColor={floodColor}
-        floodOpacity={floodOpacity}
-      />
-    </filter>
-  </SVGDefs>
-);
+}) => {
+  if (window.navigator.userAgent.includes('Edge')) {
+    // feDropShadow is not supported by Edge
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/components/svg/SvgDropShadowFilter.tsx]: I specifically wrote this as a whole if-block by itself as Edge is going Chromium in the new year, perhaps we just drop this whole if when that day comes.

File: frontend/packages/dev-console/src/components/svg/SvgDropShadowFilter.tsx [hunk 2]
```diff
@@ -19,24 +19,48 @@ const SvgDropShadowFilter: React.FC<SvgDropShadowFilterProps> = ({
...
+          y={`-${stdDeviation * 12.5}%`}
+          width={`${100 + stdDeviation * 25}%`}
+          height={`${100 + stdDeviation * 25}%`}
+        >
+          <feGaussianBlur in="SourceAlpha" stdDeviation={stdDeviation} />
+          <feOffset dx={dx} dy={dy} />
+          <feMerge>
+            <feMergeNode />
+            <feMergeNode in="SourceGraphic" />
+          </feMerge>
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/components/svg/SvgDropShadowFilter.tsx]: Your drop shadow is causing the node to look darker on edge.
I found this online that says it's equivalent of `feDropShadow`. https://docs.w3cub.com/svg/element/fedropshadow/
```
<feGaussianBlur in="alpha-channel-of-feDropShadow-in"
    stdDeviation="stdDeviation-of-feDropShadow"/>
<feOffset dx="dx-of-feDropShadow" dy="dy-of-feDropShadow"
    result="offsetblur"/>
<feFlood flood-color="flood-color-of-feDropShadow"
    flood-opacity="flood-opacity-of-feDropShadow"/>
<feComposite in2="offsetblur" operator="in"/>
<feMerge>
  <feMergeNode/>
  <feMergeNode in="in-of-feDropShadow"/>
</feMerge>
```

Comment (andrewballantyne) [reply frontend/packages/dev-console/src/components/svg/SvgDropShadowFilter.tsx]: Very well, I didn't spend a lot of time on it... Edge doesn't run smoothly, having darker shadows wasn't high on my fix list. I'll update it to be this and report back with a new screenshot.

Comment (andrewballantyne) [reply frontend/packages/dev-console/src/components/svg/SvgDropShadowFilter.tsx]: Fixed, updated the description image to showcase the result.

**Comment by andrewballantyne** [comment]
/retest

It appears to be unrelated to my fix; going to run it once more 🤞 If it fails again, I'll investigate more.

**Pull request merged**